recipes/tempo2/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "tempo2" %}
+{% set version = "2020.07.1" %}
+
+package:
+    name: "{{ name|lower }}"
```

Comment:
We are using 2 spaces instead of 4 in the recipe yaml file. Not a big deal but it is nice to have some consistency.

Reply:
I've fixed this to use two spaces now, thanks/